PayButton/paybutton-server
# [Issue #596] feat: email verification

Related to #596 


Depends on
---
- [x] #595 




Description
---
Adds email verification to the sign up process.



Test plan
---
1. Create an account, check that all steps up to verifying the email work well.
2. Check that unverified users get redirected to `/verify` when trying to access other pages.
3. Check that after verification `/verify` redirects to the dashboard.
